test-tdd_buckets.cpp

```diff
@@ -0,0 +1,11 @@
+#include "tdd_buckets.h"
+
+int main(){
+    assert(getReadingsFromRange(3,5)==4);

```

Comment:
The requirements need to be depicted in the tests clearly.
for ex. in the range 3, 3, 4, 5, 10, 11, 12 => there are 2 consecutive ranges 3-5, 10-12
num of elements in 3-5 = 4
num of elements in 10-12 = 3
So the output from production code expected is 2 ranges and count is 4,3 correspondingly.

Reply:
Hi prithamvshetty,

Thanks for the correction. I have now added the above mentioned point.

Reply:
👍